packages/components/src/components/inline-loading/_inline-loading.scss

```diff
@@ -29,6 +29,12 @@
     .#{$prefix}--loading__stroke {
       stroke-dashoffset: 99;
     }
+
+    .#{$prefix}--loading__background,
+    .#{$prefix}--loading__stroke {
+      // Ensures (r * 2 + (stroke width)) / (viewbox unit size) * (viewbox pixel size) === 14
+      r: 26.8125;
```

Comment:
it looks like Firefox still does not properly support modifying `r` with CSS?

Firefox 69 and 71

![image](https://user-images.githubusercontent.com/8265238/67106148-ac9b7f80-f18f-11e9-8464-b06a9e4145a9.png)

Chrome

![image](https://user-images.githubusercontent.com/8265238/67106264-dd7bb480-f18f-11e9-877e-e1f880b73a40.png)